.github/workflows/k8-infra-ci.yaml

```diff
@@ -64,4 +66,29 @@ jobs:
         quiet: true
         soft_fail: true
         output_format: json
-    
\ No newline at end of file
+
+  release-charts:
+    name: Release Charts
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+    - name: Run chart-releaser
+      uses: helm/chart-releaser-action@v1.5.0
+      with:
+        charts_dir: deployment
```

Comment:
charts_dir direct should be deployment/kubernetes ?

Reply:
The way chart-releaser is set up, it immediately looks into the different sub folders for the Chart.yaml instead of the specified folder. E.g. if it would specify deployment/kubernetes, it'd look in the charts and images directories for the Chart.yaml, which we don't want. We wan't the Chart.yaml file in deployment/kubernetes. Doing it this way, it looks in all the sub directories of deployment which contains kuberenetes. 

Hope this clears it up a bit.